src/requests/user.js

```diff
@@ -232,6 +232,38 @@ const search = function (userType, pageNo, pageSize, searchText, userServiceQuer
 		}
 	})
 }
+
+const listOrganization = function (organizationIds = []) {
+	return new Promise(async (resolve, reject) => {
+		const options = {
+			headers: {
+				'Content-Type': 'application/json',
+				internal_access_token: process.env.INTERNAL_ACCESS_TOKEN,
+			},
+			form: {
+				organizationIds,
+			},
+		}
+
+		const orgListUrl = userBaseUrl + endpoints.ORGANIZATION_LIST
+		try {
+			request.get(orgListUrl, options, callback)
```

Comment:
Use the generic requests functions for making this change, instead of using request package.